packages/office-ui-fabric-react/src/components/DatePicker/examples/DatePicker.Basic.Example.tsx

```diff
@@ -20,6 +19,17 @@ const DayPickerStrings: IDatePickerStrings = {
   closeButtonAriaLabel: 'Close date picker'
 };
 
+const styles = mergeStyleSets({
```

Comment:
Since both of these sets of styles are the same, it's better to use the same class for both of them and call it something generic (like `control` maybe). Same comment on the other examples.

Alternatively, you could go even simpler and just use `const controlClass = mergeStyles({...})` since there's only one class name.

Reply:
As in just renaming `styles` to `controlClass`?

Reply:
Got it! Will fix now

Reply:
That's actually kind of odd...typically `___Class` implies that it's *one* class name, not a set. It's clearer to stick with `styles` or `classNames` for sets of class names. I suggested the name `controlClass` assuming that you'd switch to `mergeStyles` which returns a single class (since there's not really a reason to use a set here).